scripts/dev_scripts/integration_tests.sh

```diff
@@ -74,6 +74,17 @@ python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
 python $COMPARE_JSON_OUT $JSON_RESULT $JSON_EXPECTED || log_fail
 
+echo -e "\n----------------------------------------------------------------------------------"
+echo "apache/maven: Analyzing the repo path, the branch name and the commit digest with dependency resolution using a CycloneDx SBOM."
+echo -e "----------------------------------------------------------------------------------\n"
+SBOM_FILE=$WORKSPACE/tests/e2e/resources/apache_maven_root_sbom.json
```

Comment:
Please move this file to `tests/dependency_analyzer/resources`.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/105/commits/1b3677a7093847f0462b857965bf96bda0b8a4a0